.travis.yml

```diff
@@ -22,8 +22,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE=partner_multi_relation
```

Comment:
in earlier versions, we had some wild patches on crucial methods (res.partner#write for example) which triggered some bugs and also had some bugs itself